### DISCUSSION TITLE
How to make `chrome-aws-lambda` work?

### DISCUSSION THREAD

**Discussion by matiadev** [vercel/community#586] [Help]
I'm using SvelteKit and I'm trying to use `chrome-aws-lambda` but I get this error after I hit the API endpoint.

```sh
Cannot find module '/var/task/puppeteer/lib/Browser'
Require stack:
- /var/task/index.js
```

I tried looking it up and reading other discussions such as https://github.com/vercel/community/discussions/124 and looking at the [vercel/og-image](https://github.com/vercel/og-image) repository but I don't see what I might be doing wrong.

```js
// endpoint

import chromium from 'chrome-aws-lambda'

export async function get() {
  const browser = await chromium.puppeteer.launch({
    args: chromium.args,
    defaultViewport: chromium.defaultViewport,
    executablePath: await chromium.executablePath,
    headless: true,
    ignoreHTTPSErrors: true
  })

  // ...
}
```

**Comment by matiadev** [comment]
After more research I figured out I had to also use `puppeteer-core` but I ran into another problem.

```js
// endpoint

import chromium from 'chrome-aws-lambda'
import puppeteer from 'puppeteer-core'

export async function get() {
  const browser = await puppeteer.launch({
    args: chromium.args,
    defaultViewport: chromium.defaultViewport,
    executablePath: await chromium.executablePath,
    headless: true,
    ignoreHTTPSErrors: true
  })

  // ...
}
```

It can't find the Chromium binary.

```js
ENOENT: no such file or directory, open '/var/bin/chromium.br'
```

I also get another error which I don't know anything about.

```sh
spawn ETXTBSY
```

**Comment by leerob** [comment]
Hey – wanted to follow up here. What are you looking to use Chromium for?

**Reply by matiadev** [reply]
Thank you! 😄

I'm using Puppeteer for social share images at [Joy of Code](https://joyofcode.xyz/) and I wanted to use Svelte for the templating but I couldn't make it work so I spruced up the [vercel/og-image](https://github.com/vercel/og-image) example and made [mattcroat/social-share-images](https://github.com/mattcroat/social-share-images).
